src/sys/resource.rs

```diff
@@ -0,0 +1,49 @@
+use std::mem;
+
+use libc::{self, c_int};
+pub use libc::{rlimit, RLIM_INFINITY};
```

Comment:
oh I was deciding not to do that since we aren't doing anything with the type. I wanted it to be easy to specify inline, eg

```
setrlimit(RLIMIT_STACK, rlimit { rlim_cur: 1024*1024, rlim_max: RLIM_INFINITY }).unwrap()
```